cluster-autoscaler-aws/components/cluster-autoscaler-aws/application.yaml

```diff
@@ -0,0 +1,34 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+  name: <CLUSTER_NAME>-cluster-autoscaler-aws
+  namespace: argocd
+  annotations:
+    argocd.argoproj.io/sync-wave: '20'
```

Comment:
there should be extra annotations 

``` bash
    kubefirst.konstruct.io/application-name: cluster-autoscaler-aws
    kubefirst.konstruct.io/source: catalog-templates
```

[Link](https://github.com/konstructio/gitops-catalog/blob/main/CONTRIBUTING.md) these changes were proposed yesterday :)